### DISCUSSION TITLE
Subscribe and display H.265 in Foxglove Studio

### DISCUSSION THREAD

**Discussion by Longtimenoc** [foxglove/community#794] [Q&A] [answered]
Can H.265 be subscribed and displayed in Foxglove Studio? If it is possible, what should I do?

**Comment by defunctzombie** [comment, marked as answer]
It is not possible. We are working on H264 support but will not have H265. If you need H265 your best bet today is to make a custom panel: https://foxglove.dev/docs/studio/extensions/getting-started
